src/components/reusable-components/searcher/form/styles.module.scss

```
@@ -0,0 +1,36 @@
+.buscador{

```

Ten presente los espaciados en el código: 
```suggestion
.buscador {
```